sale_order_type/tests/test_sale_order_type.py

```diff
@@ -68,3 +70,9 @@ def test_sale_order_confirm(self):
         for picking in sale_order.picking_ids:
             self.assertEqual(self.sale_type.invoice_state,
                              picking.invoice_state)
+
+    def test_stock_picking_create(self):
+        self.picking_out = self.picking_model.create({
```

Comment:
There's no assert here to test any condition, so the tests is useless (except possible code errors). You can just simply check that self.picking_out is really created.